install_script.sh

```diff
@@ -66,19 +56,20 @@ function install-fzf() {
 }
 
 function fix-apt_pkg() {
+    #DEPRECATED Not applies if python3.7 is not set as default
```

Comment:
maybe, show a message what explain why is deprecated, like `DEPRECATED Not applies if python3.7 is not set as default`

Reply:
Added message and confirmation to use this function